### PR TITLE
Add Pull Request Examples

### PR DESCRIPTION
**Pull request by allevin**
Per @sfdye request, here is an example to acquire Pull Requests.

### CODE REVIEW

File: doc/examples/PullRequests.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+--------------------------
+
+.. code-block:: python
+
+    # Filters for the pull requests to collect.
+    # state = 'open' | 'closed' | 'all', Default = 'open'
+    # sort = 'created' | 'updated' | popularity' | 'long-running', Default = 'created'
+    # direction = 'asc' | 'desc', 'desc' when sort = 'created" otherwise 'asc'
+    # base = Filter by base (i.e. target) branch name
+    # head = Filter by head (i.e. source) user and branch name (ex. user:ref-name)
```
Comment (sfdye) [inline doc/examples/PullRequests.rst]: In short, just a simple and working POC will do.

You can also show the output of the print result, like what I did in the other examples.

File: doc/examples/PullRequests.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    repo = g.get_repo("PyGithub/PyGithub")
+    pulls = repo.get_pulls(state='open', sort='created', base='master') 
+    pr in pulls:
+        print pr.number
```
Comment (sfdye) [inline doc/examples/PullRequests.rst]: print() to favor Python 3

**Comment by sfdye** [comment]
Also you need to add a reference in the `examples.rst`

**Comment by sfdye** [comment]
Filename in singular form (`PullRequest` not `PullRequests`), to be consistent across whole project.

**Comment by allevin** [comment]
@sfdye, I updated the example per your comments.  Hopefully this is more in line on what you are looking for.

**Comment by BarbourSmith** [comment]
Awesome work @allevin I'm gong to follow your lead once this is merged 👍 👍 💯 ❤️

**Comment by sfdye** [comment]
@allevin Thank you, as always.

**Pull request merged**